assets/airports/airportLoadList.js

```diff
@@ -243,12 +243,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
         //     name: 'Jacksonville International Airport',
         //     premium: false
         // },
-        // {
-        //     icao: 'kjfk',
-        //     level: 'hard',
-        //     name: 'John F Kennedy International Airport',
-        //     premium: false
-        // },
+        {
+            icao: 'kjfk',
+            level: 'hard',
+            name: 'John F Kennedy International Airport',
```

Comment:
Missing the period/full stop/whatever you call it  (`.`) after the F

Reply:
done